docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -735,7 +735,8 @@ to enable or disable execution of the transformation during a model conversion.
 2. Attribute `id` specifies a unique transformation string identifier. This transformation identified can be used to
 enable (disable) the transformation by setting environment variable `MO_ENABLED_TRANSFORMS` (`MO_DISABLED_TRANSFORMS`)
 with a comma separated list of `id`s. The environment variables override the value of the `enabled` attribute of the
-transformation. Optional attribute.
+transformation. Instead of using `id` attribute value you can add fully defined class name to `MO_ENABLED_TRANSFORMS` 
+(`MO_DISABLED_TRANSFORMS`) variable, `extensions.back.NonmalizeToNormalizeL2.NormalizeToNormalizeL2` for example. Optional attribute.
```

Comment:
typo in the name `NonmalizeToNormalizeL2` -> `NormalizeToNormalizeL2`

Reply:
And avoid exceeding line width of 120 characters.

Reply:
Fixed